DesktopUI2/DesktopUI2/Models/Settings/NumericSetting.cs

```diff
@@ -0,0 +1,32 @@
+﻿using DesktopUI2.Views.Settings;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace DesktopUI2.Models.Settings
+{
+  public class NumericSetting : ISetting
+  {
+    public string Type => typeof(NumericSetting).ToString();
+    public string Name { get; set; }
+    public string Slug { get; set; }
+    public string Icon { get; set; }
+    public string Description { get; set; }
+    public string Selection { get; set; }
+    public Type ViewType { get; } = typeof(NumericSettingView);
+    public string Summary { get; set; }
+
+    // Default type for Avalonia NumericUpDown is Double
+    public double Value
+    {
+      get => double.Parse(Selection);
+      set => Selection = value.ToString(CultureInfo.InvariantCulture);
+    }
```

Comment:
Seems like the user is not allowed to type text, but only numbers right?

Reply:
That's the purpose of this setting. Leverage the built in control rather than scoping casting or validation.